[2025-05-14T15:55:14.907+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Sales_Performance_ETL.Load_Data scheduled__2025-05-14T15:50:00+00:00 [queued]>
[2025-05-14T15:55:14.918+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Sales_Performance_ETL.Load_Data scheduled__2025-05-14T15:50:00+00:00 [queued]>
[2025-05-14T15:55:14.919+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2025-05-14T15:55:14.920+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2025-05-14T15:55:14.921+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2025-05-14T15:55:14.938+0000] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): Load_Data> on 2025-05-14 15:50:00+00:00
[2025-05-14T15:55:14.943+0000] {standard_task_runner.py:52} INFO - Started process 29987 to run task
[2025-05-14T15:55:14.946+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Sales_Performance_ETL', 'Load_Data', 'scheduled__2025-05-14T15:50:00+00:00', '--job-id', '131', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_Hisham_Audiputra_Suharto_DAG.py', '--cfg-path', '/tmp/tmp0bl20poc', '--error-file', '/tmp/tmplaw97xt3']
[2025-05-14T15:55:14.948+0000] {standard_task_runner.py:80} INFO - Job 131: Subtask Load_Data
[2025-05-14T15:55:15.013+0000] {task_command.py:371} INFO - Running <TaskInstance: Sales_Performance_ETL.Load_Data scheduled__2025-05-14T15:50:00+00:00 [running]> on host 2ca7303636ab
[2025-05-14T15:55:15.102+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Hisham
AIRFLOW_CTX_DAG_ID=Sales_Performance_ETL
AIRFLOW_CTX_TASK_ID=Load_Data
AIRFLOW_CTX_EXECUTION_DATE=2025-05-14T15:50:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-05-14T15:50:00+00:00
[2025-05-14T15:55:15.103+0000] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-05-14T15:55:15.104+0000] {subprocess.py:74} INFO - Running command: ['/bin/bash', '-c', 'sudo -u *** python /opt/***/scripts/Load_Data.py']
[2025-05-14T15:55:15.113+0000] {subprocess.py:85} INFO - Output:
[2025-05-14T15:55:16.809+0000] {subprocess.py:92} INFO - WARNING: An illegal reflective access operation has occurred
[2025-05-14T15:55:16.810+0000] {subprocess.py:92} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.7/site-packages/pyspark/jars/spark-unsafe_2.12-3.1.2.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2025-05-14T15:55:16.811+0000] {subprocess.py:92} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2025-05-14T15:55:16.811+0000] {subprocess.py:92} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2025-05-14T15:55:16.812+0000] {subprocess.py:92} INFO - WARNING: All illegal access operations will be denied in a future release
[2025-05-14T15:55:16.956+0000] {subprocess.py:92} INFO - :: loading settings :: url = jar:file:/home/***/.local/lib/python3.7/site-packages/pyspark/jars/ivy-2.4.0.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2025-05-14T15:55:17.024+0000] {subprocess.py:92} INFO - Ivy Default Cache set to: /home/***/.ivy2/cache
[2025-05-14T15:55:17.025+0000] {subprocess.py:92} INFO - The jars for the packages stored in: /home/***/.ivy2/jars
[2025-05-14T15:55:17.028+0000] {subprocess.py:92} INFO - org.postgresql#postgresql added as a dependency
[2025-05-14T15:55:17.029+0000] {subprocess.py:92} INFO - :: resolving dependencies :: org.apache.spark#spark-submit-parent-7f84bd42-f883-414f-9181-1f02f2e1204b;1.0
[2025-05-14T15:55:17.029+0000] {subprocess.py:92} INFO - 	confs: [default]
[2025-05-14T15:55:17.129+0000] {subprocess.py:92} INFO - 	found org.postgresql#postgresql;42.6.0 in central
[2025-05-14T15:55:17.160+0000] {subprocess.py:92} INFO - 	found org.checkerframework#checker-qual;3.31.0 in central
[2025-05-14T15:55:17.171+0000] {subprocess.py:92} INFO - :: resolution report :: resolve 139ms :: artifacts dl 3ms
[2025-05-14T15:55:17.172+0000] {subprocess.py:92} INFO - 	:: modules in use:
[2025-05-14T15:55:17.173+0000] {subprocess.py:92} INFO - 	org.checkerframework#checker-qual;3.31.0 from central in [default]
[2025-05-14T15:55:17.174+0000] {subprocess.py:92} INFO - 	org.postgresql#postgresql;42.6.0 from central in [default]
[2025-05-14T15:55:17.174+0000] {subprocess.py:92} INFO - 	---------------------------------------------------------------------
[2025-05-14T15:55:17.175+0000] {subprocess.py:92} INFO - 	|                  |            modules            ||   artifacts   |
[2025-05-14T15:55:17.175+0000] {subprocess.py:92} INFO - 	|       conf       | number| search|dwnlded|evicted|| number|dwnlded|
[2025-05-14T15:55:17.176+0000] {subprocess.py:92} INFO - 	---------------------------------------------------------------------
[2025-05-14T15:55:17.177+0000] {subprocess.py:92} INFO - 	|      default     |   2   |   0   |   0   |   0   ||   2   |   0   |
[2025-05-14T15:55:17.178+0000] {subprocess.py:92} INFO - 	---------------------------------------------------------------------
[2025-05-14T15:55:17.179+0000] {subprocess.py:92} INFO - :: retrieving :: org.apache.spark#spark-submit-parent-7f84bd42-f883-414f-9181-1f02f2e1204b
[2025-05-14T15:55:17.180+0000] {subprocess.py:92} INFO - 	confs: [default]
[2025-05-14T15:55:17.181+0000] {subprocess.py:92} INFO - 	0 artifacts copied, 2 already retrieved (0kB/6ms)
[2025-05-14T15:55:17.347+0000] {subprocess.py:92} INFO - 25/05/14 15:55:17 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-05-14T15:55:17.582+0000] {subprocess.py:92} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2025-05-14T15:55:17.583+0000] {subprocess.py:92} INFO - Setting default log level to "WARN".
[2025-05-14T15:55:17.584+0000] {subprocess.py:92} INFO - To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
[2025-05-14T15:55:29.120+0000] {subprocess.py:96} INFO - Command exited with return code 0
[2025-05-14T15:55:29.157+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=Sales_Performance_ETL, task_id=Load_Data, execution_date=20250514T155000, start_date=20250514T155514, end_date=20250514T155529
[2025-05-14T15:55:29.213+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-05-14T15:55:29.263+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
